packages/teams-js/src/public/runtime.ts

```diff
@@ -127,6 +159,61 @@ export const v1HostClientTypes = [
   HostClientType.teamsDisplays,
 ];
 
+interface IRuntimeUpgrade {
+  versionToUpgradeFrom: number;
+  upgradeToNextVersion: (previousVersionRuntime: IBaseRuntime) => IBaseRuntime;
+}
+
+/**
+ * @internal
```

Comment:
This should be hidden already since it isn't exported, but might be good to add the @hidden keyword just so no one accidentally exports it and starts writing dev documentation for it